GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -41,34 +59,41 @@ private enum AssetCatalog: String {
 extension AssetsCatalogParser {
   static let imageSetExtension = "imageset"
 
-  fileprivate func processCatalog(items: [[String: AnyObject]], withPrefix prefix: String = "") {
+  fileprivate func process(items: [[String: Any]], withPrefix prefix: String = "") -> [Entry] {
+    var result = [Entry]()
+
     for item in items {
       guard let filename = item[AssetCatalog.filename.rawValue] as? String else { continue }
       let path = Path(filename)
 
       if path.`extension` == AssetsCatalogParser.imageSetExtension {
         // this is a simple imageset
         let imageName = path.lastComponentWithoutExtension
-        addImage(named: "\(prefix)\(imageName)")
+
+        result += [Entry(name: imageName, value: "\(prefix)\(imageName)")]
       } else {
         // this is a group/folder
-        let children = item[AssetCatalog.children.rawValue] as? [[String: AnyObject]] ?? []
+        let children = item[AssetCatalog.children.rawValue] as? [[String: Any]] ?? []
 
         if let providesNamespace = item[AssetCatalog.providesNamespace.rawValue] as? NSNumber,
-            providesNamespace.boolValue {
-          processCatalog(items: children, withPrefix: "\(prefix)\(filename)/")
+          providesNamespace.boolValue {
```

Comment:
Doesn't Swift automatically bridge boolean `NSNumber` to `Bool` with `as?`? Especially since the `AnyObject -> Any` proposal got in?

I mean, we could probably do `if let providesNamespace = item[AsstCatalog.providesNamespace.rawValue] as? Bool` directly maybe? (At least it's worth a try)